### PR TITLE
Master merge from develop

### PR DESCRIPTION
**Pull request by dasunpubudumal**
Closes #

#### Changes proposed in this pull request

- Changes from develop -> master

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.1.2
+FROM ruby:3.3.4
```
Comment (yoldas) [inline Dockerfile:1]: Could you please confirm that the version is available from Docker Hub?

File: Gemfile [hunk 2]
```diff
@@ -19,7 +19,9 @@ gem 'bundler', '>= 1.3.0'
 gem 'railties', '~> 7.0.4'
 # # RAILS DEPENDENCIES
 
-gem 'sanger_warren'
+gem 'bunny', '>= 2.22.0'
+gem 'rbtree', '~> 0.4.5'
+gem 'sanger_warren', '~> 0.4.1'
```
Comment (yoldas) [inline Gemfile:24]: Is there a repo for sanger_warren ? Where does this come from ?

File: Dockerfile (hunk 1 above)
Comment (dasunpubudumal) [reply Dockerfile:1]: Yes, it is available

File: Gemfile (hunk 2 above)
Comment (dasunpubudumal) [reply Gemfile:24]: [`sanger_warren`](https://rubygems.org/gems/sanger_warren/versions/0.4.1?locale=en) is available in RubyGems 😄

**Review by yoldas — APPROVED** [review]

**Pull request merged**